mlflow/protos/model_registry.proto

```diff
@@ -236,6 +253,7 @@ enum ModelVersionStatus {
   FAILED_DELETION = 5;
 }
 
+// Note:: Experimental: This entity may change or be removed in a future release without warning.
```

Comment:
There are a couple of messages below this (`ModelVersionDetailed`, etc) - should we mark these as experimental too?

Reply:
I got all 5 entities. 
- `ModelVersion`
- `ModelVersionDetailed`
- `RegisteredModel`
- `RegisteredModelDetailed`
- `ModelVersionStatus`

remaining messages are RPC messages that do not get documented

Reply:
Oops, got it makes sense - thanks!